www/class/centreonGraph.class.php

```diff
@@ -972,8 +972,9 @@ public function createLegend()
                     if (isset($tm["warn"]) && !empty($tm["warn"]) && $tm["warn"] != 0) {
                         $this->addArgument(
                             "HRULE:" . $tm["warn"] . $tm["ds_color_area_warn"] . ":Warning  \: " .
-                            $this->humanReadable($tm["warn"], $tm["unit"]) . "\\l "
+                            $this->humanReadable($tm["warn"], $tm["unit"])
                         );
+                        $this->addArgument("COMMENT:\\l");
```

Comment:
this line may be added for critical threshold line too ?

Reply:
This line adds a newline.
As there is no more threshold under the critical one, I'm not sure we need a newline.
At least I did not found an example where it was required :)

Reply:
OK for your @kduret ? :)